.github/workflows/triage-helper.yml

```diff
@@ -0,0 +1,27 @@
+name: Issue Triage Helper
+on:
+  issues:
+    types: [labeled]
+jobs:
+  routed:
+    runs-on: ubuntu-latest
+    if: >-
+      contains(
+        fromJSON('[
+          "Status: In Progress",
+          "Status: Backlog",
+          "Status: Needs More Information"
+        ]'),
+        github.event.label.name
+      )
+    steps:
+      - name: "Remove all 'Status: *' labels"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+        run: |
+          labels_to_remove=$(gh issue view ${{ github.event.issue.number }} --json labels --jq '[.labels[].name | select(startswith("Status: ") and . != "${{ github.event.label.name }}")] | join(",")' )
+          if [[ -n "${labels_to_remove// /}" ]]; then
```

Comment:
We still want to add the new label even if there are none to remove.

Reply:
Fair point.

Reply:
Actually, thinking more on this, if there are no labels to remove that means this label is added from scratch so no need to assert it to be there?

Yeah, thinking more on this, the race condition was due to the latent `gh issue view` call which happened some time after the event was triggered. Since now we have switched to using the event payload, we get a consistent snapshot of the issue. This means we should be able to remove the `add-label` thing.

Will try this and see what happens.

Reply:
Nope, I was wrong: https://github.com/getsentry/release-tester/issues/1#event-4695107337